user/magic-wormhole-rs/template.py

```diff
@@ -0,0 +1,40 @@
+pkgname = "magic-wormhole-rs"
+pkgver = "0.7.1"
+pkgrel = 0
+build_style = "cargo"
+make_build_args = ["--workspace"]
+hostmakedepends = ["cargo-auditable"]
+makedepends = ["rust-std"]
+pkgdesc = "Rust implementation of the Magic Wormhole protocol"
+maintainer = "tulilirockz <tulilirockz@outlook.com>"
+license = "EUPL-1.2"
+url = "https://github.com/magic-wormhole/magic-wormhole.rs"
+source = f"{url}/archive/refs/tags/{pkgver}.tar.gz"
+sha256 = "c6e2acd3cccd982f449d26184d714d4cf813f51b8b75b3e36ecbb78565b3f4e8"
+_supported_shells = ["bash", "fish", "zsh"]
+# check: tests do not work because of no network connection
+options = ["!check"]
```

Comment:
```suggestion
# generates completions with host bin
options = ["!check", "!cross"]
```

Reply:
i should probably should add !cross to topgrate too then! didnt know that restricted cross compilation

Reply:
sure, yea, missed that